package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "@flatten-js/interval-tree": "^1.0.20",
     "@fortawesome/fontawesome-free": "^6.2.1",
+    "@orchidjs/sifter": "^1.0.3",
```

Comment:
We should get this for free, this is the same library tom-select uses for searching